cmd/appengine/device_aliases.go

```diff
@@ -55,6 +56,7 @@ var aliasesRemoveCmd = &cobra.Command{
 	Example: `  astartectl appengine devices aliases remove 2TBn-jNESuuHamE2Zo1anA my-alias-tag`,
 	Args:    cobra.ExactArgs(2),
 	RunE:    aliasesDeleteF,
+	Aliases: []string{"delete", "rm", "del"},
```

Comment:
Following the piece of advice highlighted by @bettio, I would say the correct verb here is remove (you're removing an alias tag from the set of aliases), so I would leave only `rm` as alias